lte/gateway/docker/services/python/Dockerfile

```diff
@@ -39,6 +39,7 @@ RUN apt-get update && apt-get install -y \
   ruby \
   sudo \
   ruby-dev \
+  docker.io \
```

Comment:
Docker in docker is usually difficult.
Are you sure, you need it?
How does it work?

Reply:
This has todo with the existing magmad implementation.
magmad manages the other services through systemd, docker or runit
In this case, we need the package but we are not running any container